ts/features/idpay/initiative/details/components/TimelineDetailsBottomSheet.tsx

```diff
@@ -199,11 +68,42 @@ const TimelineDetailsBottomSheet = () => {
   );
 };
 
+type TimelineDetailsBottomSheetConfiguration = {
+  snapPoint: number;
+  title: string;
+};
+
+type OperationWithDetailsTypeEnum =
+  | TransactionDetailOperationTypeEnum
+  | RefundOperationTypeEnum;
+
+const bottomSheetConfigurations: Record<
```

Comment:
```js
const bottomSheetI18nKeys: Record<
  OperationWithDetailsTypeEnum,
  ReturnType<typeof I18n["t"]>
> = {
  PAID_REFUND: I18n.t("idpay.initiative.operationDetails.title.refund"),
  REJECTED_REFUND: I18n.t("idpay.initiative.operationDetails.title.refund"),
  TRANSACTION: I18n.t("idpay.initiative.operationDetails.title.transaction"),
  REVERSAL: I18n.t("idpay.initiative.operationDetails.title.transaction")
};
```
as cited, something like this would be required in case of a refactor of the height sizing